src/components/vaults/community/createVault/index.tsx

```diff
@@ -0,0 +1,187 @@
+        onClose: () => {
+          navigate(getRoute(getPage(pathname), searchParams, address))
+        },
+      },
+    })
+  }, [navigate, pathname, searchParams, address])
+
+  const handleWithdrawFreezePeriodChange = useCallback((value: string) => {
+    setWithdrawFreezePeriod(value)
+  }, [])
+
+  const handleHLSSwitch = useCallback((value: boolean) => {
+    setEnableHLSVault(value)
+  }, [])
+
+  const handleSelectAssetsClick = useCallback(() => {
+    setShowMenu(true)
+  }, [])
+
+  return (
+    <CreateVaultOverlay>
+      <form className='flex flex-col space-y-6 mb-4'>
+        <div className='flex flex-col md:flex-row gap-8'>
+          <div className='flex-1 space-y-8'>
+            <Input
+              type='text'
+              // TODO: add value and onChange
+              value={''}
+              onChange={() => {}}
+              label='Vault title'
+              placeholder='Enter vault title'
+              required
+            />
+
+            <PerformanceFee />
+
+            <HLSSwitch onChange={handleHLSSwitch} name='enableHLSVault' value={enableHLSVault} />
+
+            <Input
+              type='button'
+              value={selectedAsset.label}
+              onClick={handleSelectAssetsClick}
+              label='Vault Deposit Asset'
+              suffix={<ArrowRight />}
+              required
+            />
+            <AssetSelect
+              showMenu={showMenu}
+              setShowMenu={setShowMenu}
+              assets={assets}
+              setSelectedAsset={setSelectedAsset}
+            />
+          </div>
+
+          <div className='border border-white/5' />
+
+          <div className='flex-1 space-y-8'>
+            <Input
+              type='dropdown'
+              options={options}
+              value={withdrawFreezePeriod}
+              onChange={handleWithdrawFreezePeriodChange}
+              label='Withdraw Freeze Period'
+            />
+
+            <Input
+              type='textarea'
+              // TODO: add value and onChange
+              value={''}
+              onChange={() => {}}
+              label='Description'
+              placeholder='Enter a detailed description...'
+              maxLength={240}
+              required
+            />
+            <div className='flex flex-wrap w-full'>
+              <Text size='sm' className='w-full mb-2'>
+                Details of your vault
+              </Text>
+              <Text size='xs' className='text-white/50'>
+                Setup your wallet with Stargaze or an IBC/ICNS Domain so that you can populate a
+                profile image, name and social links.
+              </Text>
+            </div>
+          </div>
+        </div>
+
+        <div className='border border-white/5' />
+
+        <div className='flex flex-wrap justify-between items-center px-4 md:px-0 gap-2'>
+          <div className='space-y-2'>
+            <Text size='base' className='w-full'>
+              $10
```

Comment:
```suggestion
              // TODO: fetch from contract
              $10
```
also please use a `<DisplayCurrency />` component here.